intents/main.py

```diff
@@ -443,3 +457,151 @@ def _list_all_dimensions_metrics(table, dimensions, metric):
             else:
                 all_dimensions.append(column)
     return (all_dimensions, all_metrics)
+
+def get_number_of_column_label(label):
```

Comment:
Add _ prefix for these internal functions

Reply:
Done